@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -651,6 +651,18 @@ export class TiltBasedClipPlanesEvaluator extends AltitudeBasedClipPlanesEvaluat
         // Near plance calculus is pretty straightforward and does not depent on camera tilt:
         const cameraAltitude = this.getCameraAltitude(camera, projection);
         viewRanges.near = cameraAltitude - this.maxElevation;
+        if (camera instanceof THREE.PerspectiveCamera) {
+            // Now we need to account for camera tilt and frustum volume, so the longest
+            // frustum edge does not instersects with sphere, it takes the worst case
+            // scenario regardless of camera tilt, so may be improved little bit with more
+            // sophisticated algorithm.
+            // Note, the fov is vertical, otherwise we would need to
+            // translate it using aspect ratio:
+            // let aspect = camera.aspect > 1 ? camera.aspect : 1 / camera.aspect;
+            const aspect = 1.0;
+            const halfFovAngle = MathUtils.degToRad((camera.fov / 2) * aspect);
+            viewRanges.near *= Math.cos(halfFovAngle);
```

Comment:
This should account for the tilt but there is no tilt angle used. Can you please elaborate a bit on the math.

Reply:
All stuff in the solutioning page.